src/reducers/versions.tsx

```diff
@@ -428,6 +431,64 @@ export const fetchVersionsList = ({
   };
 };
 
+type CreateInternalDiffParams = {
```

Comment:
This type is called `CreateInternalDiffParams` but the function is called `createInternalDiffs`. Note the `diff` in the former but `diffs` in the latter.

Reply:
ah oops, typo